### PR TITLE
[INFRA-21748] Rename Shiro master branch to main

### PR DESCRIPTION
**Pull request by fpapon**

### CODE REVIEW

File: webapp-tutorial.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ We do all of this by adding a few things to the existing `src/main/webapp/WEB-IN
 
 * The `<listener>` declaration defines a [`ServletContextListener`](http://docs.oracle.com/javaee/6/api/javax/servlet/ServletContextListener.html) that starts up the Shiro environment (including the Shiro `SecurityManager`) upon web application startup.  By default, this listener automatically knows to look for our `WEB-INF/shiro.ini` file for Shiro configuration.
 
-* The `<filter>` declaration defines the master `ShiroFilter`.  This filter is expected to filter _all_ requests into the web application so Shiro can peform necessary identity and access control operations before allowing a request to reach the application.
+* The `<filter>` declaration defines the main `ShiroFilter`.  This filter is expected to filter _all_ requests into the web application so Shiro can peform necessary identity and access control operations before allowing a request to reach the application.
```
Comment (bmarwell) [inline webapp-tutorial.md:201]: Not a branch name.

**Review by bmarwell — COMMENTED** [review]
Just a nit. Some hits are not branch names, so it doesn’t fit the linked issue. We can do this in a separate PR or in this one, whatever you like best.

File: spring-boot.md.vtl [hunk 2]
```diff
@@ -9,7 +9,7 @@ Shiro's Spring-Boot integration is the easiest way to integrate Shiro into a Spr
 Web Applications
 ---
 
-Shiro has first-class support for Spring web applications. In a web application, all Shiro-accessible web requests must go through a master Shiro Filter. This filter itself is extremely powerful, allowing for ad-hoc custom filter chains to be executed based on any URL path expression.
+Shiro has first-class support for Spring web applications. In a web application, all Shiro-accessible web requests must go through a main Shiro Filter. This filter itself is extremely powerful, allowing for ad-hoc custom filter chains to be executed based on any URL path expression.
```
[the remark this answers is not in the document]
Comment (fpapon) [reply spring-boot.md.vtl:13]: Agree that it's not about renaming branch but it's in the same feeling of replacing some words.

File: spring-framework.md.vtl [hunk 3]
```diff
@@ -60,13 +60,13 @@ That is it, now you can get the current `Subject` using:
...
 
-You can see a full example in our [samples on Github](https://github.com/apache/shiro/tree/master/samples/spring).
+You can see a full example in our [samples on Github](https://github.com/apache/shiro/tree/main/samples/spring).
 
 <a name="SpringFramework-WebApplications"></a>
 Web Applications
 ---
 
-Shiro has first-class support for Spring web applications. In a web application, all Shiro-accessible web requests must go through a master Shiro Filter. This filter itself is extremely powerful, allowing for ad-hoc custom filter chains to be executed based on any URL path expression.
+Shiro has first-class support for Spring web applications. In a web application, all Shiro-accessible web requests must go through a main Shiro Filter. This filter itself is extremely powerful, allowing for ad-hoc custom filter chains to be executed based on any URL path expression.
```
[the remark this answers is not in the document]
Comment (fpapon) [reply spring-framework.md.vtl:69]: Agree that it's not about renaming branch but it's in the same feeling of replacing some words.

File: webapp-tutorial.md (hunk 1 above)
Comment (fpapon) [reply webapp-tutorial.md:201]: Agree that it's not about renaming branch but it's in the same feeling of replacing some words.

**Comment by fpapon** [comment]
@bmarwell @bdemers Are we good with this PR?

**Review by bmarwell — APPROVED** [review]

**Comment by bmarwell** [comment]
LGTM!

**Review by bdemers — APPROVED** [review]

**Pull request merged**